glossary.asciidoc

```diff
@@ -170,6 +170,9 @@ P2WSH::
 paper wallet::
     In the most specific sense, a paper wallet is a document containing all of the data necessary to generate any number of Bitcoin private keys, forming a wallet of keys. However, people often use the term to mean any way of storing bitcoin offline as a physical document. This second definition also includes paper keys and redeemable codes.
 
+passphrase::
+    A passphrase is an optional string created by the user that serves as an additional security factor protecting the seed, even when the seed is compromised by a thief. It can also be used as a form of plausible deniability, where a chosen passphrase leads to a wallet with a small amount of funds used to distract an attacker from the “real” wallet that contains the majority of funds.
```

Comment:
Can you please try the second sentence a different way? I think "plausible deniability" may be more in circumstances where people are trying to shed responsibility for something, claiming they didn't know about it.

Reply:
I agree that the "plausible deniability" term is usually used in court trials, but both Trezor and Ledger use it in a duress/physicial attack/theft situation (see below):

https://blog.trezor.io/passphrase-the-ultimate-protection-for-your-accounts-3a311990925b
> The second and arguably even more important addition brought to the table is plausible deniability. There is no such thing as an “incorrect passphrase” and you can create an unlimited number of wallets. This can be quickly turned to your benefit when you decide to redistribute your balances to give you a “cover”.
> ...
> In a situation where you are physically threatened by burglars, border security agents, or pretty much anyone else, you can now safely give up your PIN number (which can be changed anyway). If the assailants keep you under duress and demand a passphrase, you can give out the one with the lesser amount.

https://www.ledger.com/ledger-101-part-4-advanced-security-principles
> Plausible deniability
> We have seen how the passphrase can efficiently add security to your backup. Tt can also be used for a different reason called plausible deniability.
> 
> Instead of entering your passphrase each time you need it, you can attach it to a second PIN on your Ledger device. This results in having two valid PIN codes: one will unlock the normal set of accounts, the other the alternative set of accounts.
> 
> Therefore, if ever you were asked under duress to “open and empty your hardware wallet”, you could use the first PIN code, showing the normal accounts with minimal assets. The attacker is then satisfied and leaves the scene quickly, with limited financial damage on your side.
> 
> You must however know that plausible deniability has a limit to its efficiency. If your attacker is knowledgeable about your crypto situation, or even knows about the existence of a potential passphrase, you would most likely still have to reveal the passphrase or alternative PIN code.
> 
> This is why putting yourself out of the equation by having your high value hardware wallet and backup away from your home gives you more chance of success in case of a critical security situation.

Reply:
Ok. 👍 Thanks a bunch for taking the time to look up / share this.